### ISSUE TITLE
Unlocalized Xcode Export

### ISSUE THREAD

**Issue by shnktt** [AppLovin/AppLovin-MAX-Unity-Plugin#322]
### MAX Plugin Version

6.1.2

### Unity Version

2021.2.12f1

### Device/Platform Info

iOS platform

### Current Behavior

Despite enabling the 'Localize User Tracking Usage Description' option in the Integration Manager, the Xcode project is exported without localization.

### Expected Behavior

When the 'Localize User Tracking Usage Description' is enabled, the Xcode project should be exported with appropriate localization settings applied.

### How to Reproduce

1. Enable 'Localize User Tracking Usage Description' in the Integration Manager.
2. Build the project for iOS.
3. Observe that the exported Xcode project lacks localization.

### Additional Info

In `MaxPostProcessBuildiOS.cs`, the `ShouldRemoveLocalization` method returns true.
It appears there is no instance where `AppLovinSettings.Instance.UserTrackingUsageLocalizationEnabled` is set to true, leading to this issue.

**Comment by santoshbagadi** [comment]
@shnktt  Would you mind sharing a screenshot of your Integration Manager with the settings. 

The `AppLovinSettings.Instance.UserTrackingUsageLocalizationEnable` setting gets updated for our old terms flow. The new flow uses a different settings.

**Comment by shnktt** [comment]
Thank you for your reply.
Here's a screenshot of my Integration Manager.

![screen_shot_1](https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/assets/7140448/f3084575-130c-4d49-a456-8e9fbd4e9a4c)
![screen_shot_2](https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/assets/7140448/626efc09-dc48-440c-94c4-675bda960d44)

This project was updated from version `5.11.2`, where the Consent Flow was turned on, to the current version `6.1.2`.
The exported Xcode project does not contain `AppLovinMAXResources`.

![screen_shot_3](https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/assets/7140448/0b5214fb-c459-462b-ab47-66d7bcbc0054)

**Comment by santoshbagadi** [comment]
Thank you for sharing the info. I see the issue. We'll fix it in the next release of our plugin. 

In the meantime, you may update the `MaxPostProcessBuildiOS.ShouldRemoveLocalization` method to the following to fix the issue
```
        private static bool ShouldRemoveLocalization(string localizedUserTrackingDescription)
        {
            if (string.IsNullOrEmpty(localizedUserTrackingDescription)) return true;

            var settings = AppLovinSettings.Instance;
            var internalSettings = AppLovinInternalSettings.Instance;

            return (!internalSettings.ConsentFlowEnabled || !internalSettings.UserTrackingUsageLocalizationEnabled)
                   && (!settings.ConsentFlowEnabled || !settings.UserTrackingUsageLocalizationEnabled);
        }
```

**Comment by shnktt** [comment]
Thank you for providing the solution!
I'm looking forward to the next release of the plugin.

**Comment by santoshbagadi** [comment]
We've released [version 6.2.0](https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/releases/tag/release_6_2_0) of our plugin that fixes this issue.
